### PR TITLE
Remove manifest parsing

### PR DESCRIPTION
**Pull request by forbesjo**

### CODE REVIEW

File: test/dashjs.test.js [hunk 1]
```diff
@@ -78,11 +78,10 @@
                   'src and manifest key system options are merged');
               },
```
Comment (forbesjo) [inline test/dashjs.test.js]: `retrieveManifest` test is no longer necessary

File: src/js/videojs-dash.js [hunk 2]
```diff
@@ -57,13 +30,13 @@ var dashjs = require('dashjs');
     // again later once everything is setup
     tech.isReady_ = false;
 
+    if (Html5DashJS.updateSourceData) {
```
Comment (gesinger) [inline src/js/videojs-dash.js]: Is updateSourceData our "generic" before function that anyone can hook into?

Comment (forbesjo) [inline src/js/videojs-dash.js]: Yes, I'll add it to the README

File: src/js/videojs-dash.js [hunk 3]
```diff
@@ -81,66 +54,14 @@ var dashjs = require('dashjs');
...
-  Html5DashJS.prototype.initializeDashJS = function (manifest, err) {
-    var manifestProtectionData = {};
-
-    if (err) {
-      Html5DashJS.showErrors(this.elParent_);
-      this.tech_.triggerReady();
-      this.dispose();
-      return;
-    }
+    this.mediaPlayer_.setAutoPlay(false);
```
Comment (BrandonOCasey) [inline src/js/videojs-dash.js]: don't we still have to wrap this in `if (!options.autoplay)` so that we can autioplay? or is this handled elsewhere?

Comment (forbesjo) [inline src/js/videojs-dash.js]: This is handled in video.js https://github.com/videojs/video.js/blob/master/src/js/player.js#L775

File: src/js/videojs-dash.js (hunk 2 above)
Comment (forbesjo) [inline src/js/videojs-dash.js]: README has been updated

File: test/dashjs.test.js (hunk 1 above)
Comment (forbesjo) [inline test/dashjs.test.js]: Removed `retrieveManifest` mock

File: src/js/videojs-dash.js (hunk 3 above)
Comment (BrandonOCasey) [inline src/js/videojs-dash.js]: okay it may be good to add a comment about it being handled in video.js just so it does not confuse anyone

**Comment by BrandonOCasey** [comment]
LGTM

**Pull request merged**